extensions/amp-ad-network-doubleclick-impl/0.1/test/test-doubleclick-rtc.js

```diff
@@ -294,6 +297,7 @@ describes.realWin('DoubleClick Fast Fetch RTC', {amp: true}, env => {
         'data-multi-size-validation': macros['data-multi-size-validation'],
         'data-override-width': macros['data-OVERRIDE-width'],
         'data-override-height': macros['data-override-HEIGHT'],
+        'data-json': JSON.stringify(json),
```

Comment:
does your test coverage handle the case where data-json exists but without targeting key?

Reply:
added test